overview/models.mdx

```diff
@@ -8,41 +8,38 @@ description: "Complete list of available models on Venice AI platform"
 | Model Name | Model ID | Price (in/out) | Context Limit | Function Calling | Reasoning | Traits |
 |------------|----------|----------------|---------------|------------------|-----------|--------|
 | Venice Uncensored 1.1 | `venice-uncensored` | `$0.50 / $2.00` | 32,768 | ✕ | ✕ | — |
-| Venice Reasoning | `qwen-2.5-qwq-32b` | `$0.50 / $2.00` | 32,768 | ✕ | ✔ | — |
+| Venice Reasoning (D) | `qwen-2.5-qwq-32b` | `$0.50 / $2.00` | 32,768 | ✕ | ✔ | — |
 | Venice Small | `qwen3-4b` | `$0.15 / $0.60` | 40,960 | ✔ | ✔ | — |
 | Venice Medium (3.2 beta) | `mistral-32-24b` | `$0.50 / $2.00` | 131,072 | ✔ | ✕ | — |
 | Venice Medium (3.1) | `mistral-31-24b` | `$0.50 / $2.00` | 131,072 | ✔ | ✕ | default_vision |
 | Venice Large 1.1 | `qwen3-235b` | `$1.50 / $6.00` | 131,072 | ✔ | ✔ | — |
 | Llama 3.2 3B | `llama-3.2-3b` | `$0.15 / $0.60` | 131,072 | ✔ | ✕ | fastest |
 | Llama 3.3 70B | `llama-3.3-70b` | `$0.70 / $2.80` | 65,536 | ✔ | ✕ | default |
-| Llama 3.1 405B | `llama-3.1-405b` | `$1.50 / $6.00` | 65,536 | ✕ | ✕ | most_intelligent |
-| Dolphin 72B | `dolphin-2.9.2-qwen2-72b` | `$0.70 / $2.80` | 32,768 | ✕ | ✕ | most_uncensored |
-| Qwen 2.5 VL 72B | `qwen-2.5-vl` | `$0.70 / $2.80` | 32,768 | ✕ | ✕ | — |
-| Qwen 2.5 Coder 32B | `qwen-2.5-coder-32b` | `$0.50 / $2.00` | 32,768 | ✕ | ✕ | default_code |
-| DeepSeek R1 671B | `deepseek-r1-671b` | `$3.50 / $14.00` | 131,072 | ✕ | ✔ | default_reasoning |
+| Llama 3.1 405B (D) | `llama-3.1-405b` | `$1.50 / $6.00` | 65,536 | ✕ | ✕ | most_intelligent |
+| Dolphin 72B (D) | `dolphin-2.9.2-qwen2-72b` | `$0.70 / $2.80` | 32,768 | ✕ | ✕ | most_uncensored |
+| Qwen 2.5 VL 72B (D) | `qwen-2.5-vl` | `$0.70 / $2.80` | 32,768 | ✕ | ✕ | — |
+| Qwen 2.5 Coder 32B (D) | `qwen-2.5-coder-32b` | `$0.50 / $2.00` | 32,768 | ✕ | ✕ | default_code |
+| DeepSeek R1 671B (D) | `deepseek-r1-671b` | `$3.50 / $14.00` | 131,072 | ✕ | ✔ | default_reasoning |
 | DeepSeek Coder V2 Lite | `deepseek-coder-v2-lite` | `$0.50 / $2.00` | 131,072 | ✕ | ✕ | — |
 
-*Pricing is per million tokens (input / output)*
+*Pricing is per 1M tokens (input / output)*
 
 ### Popular Text Models
 `qwen3-235b` Venice Large 1.1 - Most powerful flagship model  
```

Comment:
Maybe consider adding something like, " ,advanced reasoning when using thinking mode".

This 1) implies the thinking and non-thinking modes and 2) uses the same text from DS R1 implying that with thinking mode it's a good replacement.